crates/eframe/src/native/wgpu_integration.rs

```diff
@@ -544,7 +546,8 @@ impl WgpuWinitRunning {
 
             {
                 crate::profile_scope!("set_window");
-                if let Err(err) = pollster::block_on(painter.set_window(viewport_id, Some(window)))
+                if let Err(err) =
+                    pollster::block_on(painter.set_window(viewport_id, Some(window.clone())))
```

Comment:
nit: it is often nicer to use `Arc::clone` to explicitly show what we're doing

```suggestion
                    pollster::block_on(painter.set_window(viewport_id, Some(Arc::clone(window))))
```